app/src/main/java/com/spendster/presentation/signUp/SignUpActivity.java

```diff
@@ -21,16 +21,18 @@ protected void onCreate(Bundle savedInstanceState) {
         final SignUpPresenter signUpPresenter = new SignUpPresenter(this, new FakeSignUpModel());
         final Button btnContinue = findViewById(R.id.btnContinue);
         final EditText etEmail = findViewById(R.id.etEmail);
+        final EditText etUsername = findViewById(R.id.etUserName);
         final EditText etPassword = findViewById(R.id.etPassword);
         final EditText etRetypePassword = findViewById(R.id.etRetypePassword);
 
         btnContinue.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
                 String email = etEmail.getText().toString();
```

Comment:
@SimonMaha please extract it into a function. Also, trim string by removing all invalid symbols. For example, empty symb.